aten/src/ATen/native/cuda/TensorFactories.cu

```diff
@@ -44,7 +44,7 @@ Tensor& eye_out_cuda(Tensor& result, int64_t n, int64_t m) {
 
 Tensor empty_cuda(IntList size, const TensorOptions& options) {
   AT_ASSERT(options.backend() == at::Backend::CUDA);
-  AT_ASSERT(!options.is_variable());  // is_variable should have been 'unpacked'
+  AT_ASSERT(!options.is_variable());  // is_variable should have been 'unpacked'  // TODO: remove this when Variable and Tensor are merged
```

Comment:
same question as above.

Reply:
I believe there is actually no clear need to check this since only `options.dtype()` is used in this function.